pkg/networkmanager/firewall.jsx

```diff
@@ -918,13 +913,6 @@ export class Firewall extends React.Component {
             );
         }
 
-        var services = [...this.state.firewall.enabledServices].map(id => {
-            const service = this.state.firewall.services[id];
-            service.name = service.name || id;
-            return service;
-        });
-        services.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
But shouldn't we keep the sorting by ID?

Reply:
the services variable here was never used, so I just removed it entirely.

I think the services are sorted alphabetically per default.